sdk/azidentity/chained_token_credential.go

```diff
@@ -62,36 +64,56 @@ func (c *ChainedTokenCredential) GetToken(ctx context.Context, opts policy.Token
 		token, err = cred.GetToken(ctx, opts)
 		var credErr credentialUnavailableError
 		if errors.As(err, &credErr) {
-			errList = append(errList, credErr)
+			errList = append(errList, newCredentialUnavailableError(extractCredentialName(cred), err.Error()))
```

Comment:
can do! Thank you, I like this 🙂

Reply:
Now we have
```go
var credErr credentialUnavailableError
if errors.As(err, &credErr) {
    errList = append(errList, ...)
}
return nil, err
```

i.e. return the first `credentialUnavailableError` instead of trying the next credential.

Reply:
my bad!

Reply:
Fixed it 🙂